odxtools/diagdatadictionaryspec.py

```diff
@@ -29,37 +30,55 @@
 
 @dataclass
 class DiagDataDictionarySpec:
+    admin_data: Optional[AdminData]
     dtc_dops: NamedItemList[DtcDop]
+    env_data_descs: NamedItemList[EnvironmentDataDescription]
     data_object_props: NamedItemList[DataObjectProperty]
     structures: NamedItemList[BasicStructure]
-    end_of_pdu_fields: NamedItemList[EndOfPduField]
     static_fields: NamedItemList[StaticField]
     dynamic_length_fields: NamedItemList[DynamicLengthField]
-    tables: NamedItemList[Table]
-    env_data_descs: NamedItemList[EnvironmentDataDescription]
-    env_datas: NamedItemList[EnvironmentData]
+    #dynamic_endmarker_fields: NamedItemList[DynamicEndmarkerField]
+    end_of_pdu_fields: NamedItemList[EndOfPduField]
     muxs: NamedItemList[Multiplexer]
+    env_datas: NamedItemList[EnvironmentData]
     unit_spec: Optional[UnitSpec]
+    tables: NamedItemList[Table]
     sdgs: List[SpecialDataGroup]
 
     def __post_init__(self) -> None:
         self._all_data_object_properties: NamedItemList[DopBase] = NamedItemList(
             chain(
                 self.dtc_dops,
+                self.env_data_descs,
                 self.data_object_props,
                 self.structures,
-                self.end_of_pdu_fields,
                 self.static_fields,
                 self.dynamic_length_fields,
-                self.env_data_descs,
-                self.env_datas,
+                #self.dynamic_endmarker_fields,
```

Comment:
commented out code?

Reply:
hint for me that this is still missing, and where this attribute ought to go once I implement dynamic endmarker fields...